packages/node_modules/@ciscospark/react-container-activity-list/src/selectors.js

```diff
@@ -6,11 +6,11 @@ import _ from 'lodash';
 
 import {ITEM_TYPE_ACTIVITY, ITEM_TYPE_DAY_SEPARATOR, ITEM_TYPE_NEW_MESSAGE_SEPARATOR} from '@ciscospark/react-component-activity-list';
 
-const getActivities = (state) => state.conversation.activities.toArray();
-const getAvatars = (state) => state.avatar.get(`items`).toJS();
-const getCurrentUser = (state) => state.user.get(`currentUser`);
+const getConversationActivities = (state) => state.conversation.activities;
```

Comment:
Did you see a big change in load times when moving the immutable deconstructors into the actual sector?